Update `Connecting Wasabi Wallet` to Wasabi 2.0 (#1180)
btcpayserver/btcpayserver-doc: docs/WasabiWallet.md

## Quick Setup

1. Create a new wallet in Wasabi
2. In Wasabi, copy the **Extended Account Public Key** at `Wallet Info`.
3. In BTCPay Server, Store > Settings > Wallet > Setup > Connect an existing wallet > Enter extended public key
4. In Wasabi, generate a new address at `Receive`.
5. Confirm that the addresses in Wasabi and BTCPay Server match.

## Step by Step

> At first launch of Wasabi, the `Add wallet` dialog will be opened automatically.

Reviewer: ```suggestion
At the first launch of Wasabi, the `Add wallet` dialog will be opened automatically.
```

Select `Create new wallet` to generate a new wallet.

![Wasabi Add Wallet](./img/Wasabi/WasabiAddWallet.png "Wasabi Add Wallet")

Give your wallet a name, for example `BTCPay Server Wallet`.

![Wasabi Add Wallet Name](./img/Wasabi/WasabiAddWalletWalletName.png "Wasabi Add Wallet Name")

Write down the Recovery Words in the correct order.

![Wasabi Add Wallet Recovery Words](./img/Wasabi/WasabiAddWalletRecoveryWords.png "Wasabi Add Wallet Recovery Words")

Confirm 3 of the 12 recovery words.
This is a quick test to make sure that you have written them down.

![Wasabi Add Wallet Confirm Recovery Words](./img/Wasabi/WasabiAddWalletConfirmRecoveryWords.png "Wasabi Add Wallet Confirm Recovery Words")

Add a password.
The password is used as a passphrase and it cannot be changed later on.

:::danger Both The Recovery words AND the password are needed to recover this wallet
Make sure to have a backup of the recovery words and the password. 
:::

![Wasabi Add Wallet Add Password](./img/Wasabi/WasabiAddWalletAddPassword.png "Wasabi Add Wallet Add Password")

> **IMPORTANT NOTE:** Write down your recovery words in the order you see them on the screen. Write them down a piece of paper and store it somewhere secure. Take your time and triple check each word. Do not store your seed in a digital format (photograph, text document). Whoever has the access to your seed and your password can access your funds. Make sure you have a proper back up of both the Recovery Words and the Password.

Reviewer: ```suggestion
**IMPORTANT NOTE:** Write down your recovery words in the order you see them on the screen. Write them down a piece of paper and store them somewhere secure. Take your time and triple-check each word. Do not store your seed in a digital format (photograph, text document). Whoever has access to your seed and your password can access your funds. Make sure you have a proper backup of the Recovery Words and the Password.
```

Select a Coinjoin Strategy.
Wasabi automatically coinjoins all your funds. 

> In case you really don't want to coinjoin your funds, you can disable coinjoin later on by disabling `Automatically start coinjoin` in the Coinjoin Settings.

Reviewer: ```suggestion
If you don't want to coinjoin your funds, you can disable coinjoin later by disabling `Automatically start coinjoin` in the Coinjoin Settings.	 
```

### Connecting Wasabi to BTCPay Server Full Node (If you're self-hosting BTCPay)

> After the wallets are connected, it is highly-recommended to **connect Wasabi Wallet to your full node in BTCPay**. The process is easy, but can only be done if you self-host BTCPay and are logged in as `Admin`. Tor has to be enabled in BTCPay (it is enabled by default). This process enhances privacy even further.

Reviewer: ```suggestion
After the wallets are connected, it is highly recommended to **connect Wasabi Wallet to your full node in BTCPay**. The process is easy but can only be done if you self-host BTCPay and are logged in as `Admin`. Tor has to be enabled in BTCPay (it is enabled by default). This process enhances privacy even further.
```

In BTCPay, go Server Settings > Services > **Full node P2P > See Information**.
On the BTCP-P2P page, click on the `Show Confidential QR Code`. Bellow the QR Code, there's a link `See QR Code information by clicking here`, so click on the link to reveal your string. Copy the string but remove `bitcoin-p2p://` part.

In Wasabi, go to the Bitcoin tab at `Settings` and paste the endpoint in `Bitcoin P2P Endpoint`.

Restart Wasabi to apply the changes.

### Configuring the Gap Limit in Wasabi

> At the SearchBar on top, click on `Wallet Folder`. Shortly the `json` file will be shown in a sub-folder. Open that file with a text editor like notepad.

Reviewer: ```suggestion
At the search bar on top, click on `Wallet Folder`. Shortly the `json` file will be shown in a sub-folder. Open that file with a text editor like notepad.
```
